README.md

```diff
@@ -10,6 +10,15 @@ see [cabal-node](https://github.com/cabal-club/cabal-node) for the underlying da
 chat with us:  
 `npx cabal --key cabal://7d99b453506b9743bf5e71fe749f66c814d7cd9388a5d394a27eed4c5640302b`
 
+## Installation
+
+```
+git clone https://github.com/cabal-club/cabal-client.git
```

Comment:
not sure if this should be the `https` or `git` url... i think `https` is more common and easier for people who don't have their keys set up. I figure folks with their github keys set up know to use the `git` url instead.